### DISCUSSION TITLE
What was Marge Simpson's - from the TV show The Simpsons - maiden name?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#399] [Galaxy Trivia] [answered]
**Hint:** Like the famous Belgium sheepdog breed

**Comment by peterbaccaro** [comment]
**Bouvier** was the maiden name of Marge Simpson from the TV show The Simpsons.

**Comment by Roberto-Larivera** [comment]
Marjorie Jacqueline "Marge" Simpson (née Bouvier)

**Comment by thuongtruong1009** [comment, marked as answer]
Marge Simpson's maiden name is Bouvier.
